Description: Changed /etc/monitrc references to
 /etc/monit/monitrc (Closes:#244646).
Author: Stefan Alfredsson <alfs@debian.org>
Forwarded: not-needed

---
 monit.1 |   10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

--- a/monit.1
+++ b/monit.1
@@ -313,7 +313,7 @@
 .IX Header "HOW TO MONITOR"
 Monit is configured and controlled via a control file called
 \&\fBmonitrc\fR. The default location for this file is ~/.monitrc. If
-this file does not exist, Monit will try /etc/monitrc, then
+this file does not exist, Monit will try /etc/monit/monitrc, then
 \&\f(CW@sysconfdir\fR@/monitrc and finally ./monitrc.
 .PP
 A Monit control file consists of a series of service entries and
@@ -469,7 +469,7 @@
 .PP
 .Vb 2
 \&  # Run Monit in standard run\-levels
-\&  mo:2345:respawn:/usr/local/bin/monit \-Ic /etc/monitrc
+\&  mo:2345:respawn:/usr/sbin/monit \-Ic /etc/monitrc
 .Ve
 .PP
 After you have modified init's configuration file, you can run
@@ -1783,7 +1783,7 @@
 Example:
 .PP
 .Vb 2
-\& check file monit.bin with path "/usr/local/bin/monit"
+\& check file monit.bin with path "/usr/sbin/monit"
 \&       if failed permission 0555 then unmonitor
 .Ve
 .PP
@@ -3799,9 +3799,9 @@
 \&\fI~/.monitrc\fR  
    Default run control file
 .PP
-\&\fI/etc/monitrc\fR
+\&\fI/etc/monit/monitrc\fR
    If the control file is not found in the default 
-   location and /etc contains a \fImonitrc\fR file, this
+   location and /etc/monit/ contains a \fImonitrc\fR file, this
    file will be used instead.
 .PP
 \&\fI./monitrc\fR  
